cityofaustin/atd-moped
# Remove signal dashboard page

## Associated issues

## Testing
**URL to test:** 


https://deploy-preview-1212--atd-moped-main.netlify.app/moped/dashboard

**Steps to test:**

Check out the dashboard. The link to the signal view is gone. Also confirm that this redirects you back to the projects. 

https://deploy-preview-1212--atd-moped-main.netlify.app/moped/dashboard/views/signal-projects


---
#### Ship list
